libs/hooks/open-telemetry/babel.config.json

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["minify"]
+  "presets": [["minify", { "builtIns": false }]]
```

Comment:
What does this do?

Reply:
I mentioned it here: https://github.com/open-feature/js-sdk-contrib/pull/142#discussion_r1014549149